shared/js/background/utils.es6.js

```diff
@@ -38,6 +38,17 @@ export async function sendTabMessage (id, message, details) {
     }
 }
 
+export async function sendAllTabsMessage (message, details) {
+    try {
+        const tabs = await browser.tabs.query({})
```

Comment:
I don't mind the intermediate variable, but since we're only assigning it to get `tabId`, I like your suggestion.

About the try/catch though, since `await browser.tabs.query({})` could throw an error, I think it's better to ensure a catch for it. Or you think it should be fine?

Reply:
@kzar Can we keep these changes as it is?

Reply:
Yea, I don't feel too strongly about it.